### PR TITLE
Release history html prototype

### LINKED ISSUE #2: Firebase SDK Dashboard: SDK release history page
FYI @shrutip90 @TMazenge

### LINKED ISSUE #5: Firebase SDK Dashboard: Directory Structure
Initialize the directory structure in the github repo

### PR DESCRIPTION
**Pull request by eriyezab**
Closes #2.
Created the html skeleton with example data for the release history page
Initializes front end directory structure.
Adds some dependencies to pom.xml.

### CODE REVIEW

**Review by TMazenge — APPROVED** [review]
LGTM

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.google.capstone</groupId>
```
Comment (ankitaj224) [inline pom.xml]: Have a generic group id specific to Firebase dashboard

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      <groupId>com.google.code.gson</groupId>
+      <artifactId>gson</artifactId>
+      <version>2.8.6</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.google.appengine</groupId>
+      <artifactId>appengine-api-1.0-sdk</artifactId>
+      <version>1.9.59</version>
+    </dependency>
```
Comment (ankitaj224) [inline pom.xml:38]: new line between dependencies.

File: target/sdktiktok-1/release_history.html [hunk 3]
```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```
Comment (ankitaj224) [inline target/sdktiktok-1/release_history.html]: Do you want to check this in?

**Review by ankitaj224 — CHANGES_REQUESTED** [review]
I am unsure if you want to check in files under target/. I presume not . PTAL.

**Review by ankitaj224 — COMMENTED** [review]
Add a directory for images/

File: src/main/webapp/enroll.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+            </div>
+        </div>
+    </div>
+
+    <hr>
+   
+   <div class="border">
+        <div class="container" >
+        <h2>SDK Library Detail</h2>
+        <form action="/action_page.php" class="text-center">
```
Comment (ankitaj224) [inline src/main/webapp/enroll.html]: php?

**Review by ankitaj224 — APPROVED** [review]

**Pull request merged**